src/middlewares/isAdmin.js

```diff
@@ -1,21 +0,0 @@
-const jwt = require('jsonwebtoken');
```

Comment:
why are u deleting the isAdmin file?

Reply:
i am not deleting its replaced with the isLogged but now it should be fine both

Reply:
I think @felmez you should write both the isAdmin and isLogedIn files in their latest state in this branch and then push

Reply:
already did this, thanks